data/geany-3.0.css

```diff
@@ -1,10 +0,0 @@
-@import "geany.css";
-
-/* make close button on the editor's tabs smaller */
-#geany-close-tab-button {
-	-GtkWidget-focus-padding: 0;
-	-GtkWidget-focus-line-width: 0;
-	-GtkButton-default-border: 0;
-	-GtkButton-default-outside-border: 0;
-	-GtkButton-inner-border: 0;
-}
```

Comment:
Using `~/.config/gtk-3.0/gtk.css`, how do you make all the gtkpaned splitters wider or make the statusbar background colour pink, just in Geany without affecting other applications? Using the global gtk.css would only allow you to tweak Geany's named IDs without affecting other applications, any other changes will affect all GTK+ apps.

The only other way is quite a bit complicated, involving custom GTK+ themes applied to Geany alone using environment variables to override the theme, AFAICT.